inst/sql/modified_update/target_pop_routine.sql

```diff
@@ -0,0 +1,22 @@
+SELECT demographic_statistic.demographic_source,
+       demographic_statistic.age_from,
+       demographic_statistic.country,
+       demographic_statistic.year,
+       demographic_statistic.value
+  FROM demographic_source
+  JOIN demographic_statistic
+    ON demographic_statistic.demographic_source = demographic_source.id
+  JOIN gender
+    ON gender.id = demographic_statistic.gender
+  JOIN demographic_statistic_type
+    ON demographic_statistic_type.id =
+         demographic_statistic.demographic_statistic_type
+ WHERE gender.code = 'both'
+   AND demographic_statistic_type.code = 'int_pop'
+-- This will need expanding if we need further detail but this will
+-- hopefully do for now (and will make the queries a little easier to
+-- deal with)
+   AND age_from IN (0, 1, 9)
```

Comment:
We may want to use age_from and age_to from the coverage table in the future. Because, even for routine, age range can vary for specific countries. (Only a few countries for HPV).

And HPV vaccine is for female only. There are a few campaigns targeting either female or male (not very big campaigns). So, maybe we can improve the demographic part in the future.

Reply:
Can you please create a new youtrack issue explaining this a bit further?